Controller/ImagineController.php

```diff
@@ -143,7 +143,7 @@ private function createRedirectResponse(\Closure $url, string $path, string $fil
             return new RedirectResponse($url(), $this->controllerConfig->getRedirectResponseCode());
         } catch (NotLoadableException $exception) {
             if (null !== $this->dataManager->getDefaultImageUrl($filter)) {
-                return new RedirectResponse($this->dataManager->getDefaultImageUrl($filter));
+                return new RedirectResponse($this->dataManager->getDefaultImageUrl($filter), $this->controllerConfig->getRedirectResponseCode());
```

Comment:
I do not think it's a good idea. You make a 301 redirect to the default image if the requested image is not found. If the picture is added, the user will not see it because his request has been cached.